apps/am/am-org-role-mapping-service/aat.yaml

```diff
@@ -5,8 +5,33 @@ metadata:
 spec:
   values:
     java:
+      keyVaults:
+        am:
+          secrets:
+            - name: app-insights-connection-string
+              alias: app-insights-connection-string
+            - name: am-org-role-mapping-service-s2s-secret
+              alias: AM_ORG_ROLE_MAPPING_SERVICE_SECRET
+            - name: role-assignment-service-LD-SDK-KEY
+              alias: LD_SDK_KEY
+            - name: orm-IDAM-CLIENT-ID
+              alias: ORM_IDAM_CLIENT_ID
+            - name: orm-IDAM-CLIENT-SECRET
+              alias: ORG_ROLE_MAPPING_IDAM_CLIENT_SECRET
+            - name: orm-IDAM-ADMIN-SECRET
+              alias: ORG_ROLE_MAPPING_IDAM_ADMIN_PASSWORD
+            - name: caseworker-topic-primary-send-listen-shared-access-key
+              alias: AMQP_CRD_SHARED_ACCESS_KEY_VALUE
+            - name: org-role-mapping-service-POSTGRES-PASS-V15
+              alias: ORG_ROLE_MAPPING_DB_PASSWORD
+            - name: judicial-topic-primary-send-listen-shared-access-key
+              alias: AMQP_JRD_SHARED_ACCESS_KEY_VALUE
       environment:
         AM_INFO: true
         APPLICATION_LOGGING_LEVEL: INFO
         SPRING_DATASOURCE_HIKARI_CONNECTIONTIMEOUT: 10000
         TESTING_SUPPORT_ENABLED: true
+        JUDICIAL_REF_APP_V2_ACTIVE: true
+        JUDICIAL_REF_APP_V2_FILTER_AUTHS_BY_APP_ID: true
```

Comment:
The two V2 flags were retired in #28917.

Reply:
these have been removed